modepy/shapes.py

```diff
@@ -276,6 +277,31 @@ class Face:
     map_to_volume: Callable[[np.ndarray], np.ndarray]
 
 
+def face_normal(face: Face, normalize=True) -> np.ndarray:
+    """
+    .. versionadded :: 2021.2.1
+    """
```

Comment:
Maybe a short description of what's going on for those that might be a bit rusty with geometric algebra :sweat_smile: (me)

Reply:
2f519b9

Reply:
84190bb is prettier.

Reply:
Thanks!